packages/block-library/src/navigation/edit/index.js

```diff
@@ -833,6 +833,33 @@ function Navigation( {
 		);
 	}
 
+	const navigationMenuSelectorInstance = (
```

Comment:
Yea, I'd rather reafactor _all_ the usages of `NavigationMenuSelector`

Reply:
The other uses pass different properties, so that's a bit more complex.